packages/cli/test/commands/new.test.ts

```diff
@@ -137,9 +137,14 @@ describe("`new` command", () => {
       })
 
       const newAppDir = fs.mkdtempSync(path.join(tempDir, "full-install-"))
-      await whileStayingInCWD(() => New.run([newAppDir, "--skip-upgrade"]))
-
-      expect(getStepsFromOutput()).toStrictEqual([`cd ${newAppDir}`, "blitz dev"])
+      await whileStayingInCWD(() => New.run([newAppDir, "--pnpm", "--skip-upgrade"]))
+
+      expect(getStepsFromOutput()).toStrictEqual([
+        `cd ${newAppDir}`,
+        "pnpm install",
+        "blitz prisma migrate dev (when asked, you can name the migration anything)",
```

Comment:
There were not all the steps. I think this test was a false positive because of the `dryRun` condition bug.